### DISCUSSION TITLE
Allow Issue subscriptions for

### DISCUSSION THREAD

**Discussion by aaronsteers** [community/community#57046] [Projects and Issues]
### Select Topic Area

Product Feedback

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

I was extremely excited to see a "Status" option in the notifications pane for GitHub Issues. 

> <img width="322" alt="image" src="https://github.com/community/community/assets/18150651/3b016402-6556-486a-8e63-36e9f56c1565">

However, when I drilled down, it didn't give the feature that I would expect - which would be to _also_ ping me if a maintainer posts a comment. This would mean filtering out comments from non-maintainers - or at least filtering out non-contributors' comments. The non-contributors and non-maintainers comments are the ones who will be asking "Is this being built yet?" or "I want this feature too!" - and by definition those very often are not pushing the conversation forward.

To further make this "Status" option useful in the notifications pane, my proposal would be to add "Comments from Maintainers" to this custom list:

> <img width="448" alt="image" src="https://github.com/community/community/assets/18150651/a528132c-c068-44e1-84cf-01dda21e884f">

And then the "Status" option in the first screenshot above could optionally default to include "Comments from Maintainers" as well as close events.

The reason I'm proposing it is that for most/all very large and important long-term issues, we (the community) want to know if there are updates to the issue - even if those are not "closure" events.

Thanks for all that you are doing. Hopefully this feature proposal resonates with others as well. 🙏

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by ggnall** [comment]
I would love to be able to subscribe to updates within a project. Our projects represent teams that may work across repos. I'd love to subscribe specifically to updates for all issues within a project, rather than drinking from the firehouse of the 3 repos my project issues are a part of.
